andela/taps-client
# #161059614 Member should be able to create permission request

#### What does this PR do?
Allow members to make an admin permission request 
#### Description of Task to be completed?
This allows an authenticated user with `member` role to make a request to have an admin privilege on the application
#### How should this be manually tested?
- Login to the app on https://ghoulies-taps-client-pr-21.herokuapp.com/ using Andela email
- You will see a `Request admin privilege` button on the navigation bar
- Click on the button
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#161059614](https://www.pivotaltracker.com/story/show/161059614)
#### Screenshots or gifs (if appropriate)
<img width="784" alt="screen shot 2018-10-11 at 3 06 32 pm" src="https://user-images.githubusercontent.com/31155397/46809857-53706300-cd67-11e8-97ea-92f9beecf757.png">
<img width="871" alt="screen shot 2018-10-11 at 3 06 42 pm" src="https://user-images.githubusercontent.com/31155397/46809858-53706300-cd67-11e8-971b-c4917a3a1a46.png">

#### Questions:
N/A
